pyproject.toml

```diff
@@ -71,7 +71,7 @@ dask = [
     "woodwork[dask] >= 0.23.0",
 ]
 spark = [
-    "woodwork[spark] >= 0.23.0",
+    "woodwork[spark] >= 0.23.0, <0.28.0",
```

Comment:
pinning woodwork[spark] below woodwork 0.28.0 because woodwork 0.28 requires pandas > 2.0, and we have [test failures](https://github.com/alteryx/featuretools/actions/runs/7832364459/job/21370894618#step:12:467) that come from doing `ps.to_datetime(lti_df["last_time"])` when pandas is above 2.0. I haven't figured out what is causing those failures, so I decided to keep pandas pinned below 2.0 as well as maintain the rest of the spark dependencies for the spark instillation for now.